.github/workflows/ci-pull.yml

```diff
@@ -7,7 +7,7 @@ on:
   #pull_request: TODO enable after test passes
   #  branches: [ "main" ]
   push:
-    branches: [ "actions"]  #Allows testing Actions changes in a fork
+    branches: [ "issue_2_3"]  #Allows testing Actions changes in a fork
```

Comment:
Because you changed this to match your branch, it does work for testing currently. I would change it back before delivering this, but you can leave it as is to confirm the Actions build:
https://github.com/revijay/sample-cloudant/actions/runs/11821762255/job/32937244453

Reply:
Okay Mark. I will leave the branch name as is to test and confirm the build is success after merging pom.xml changes from Sweetty's commit. Now the build is failing due to compilation errors because of missing dependencies.